openreview/venue/venue.py

```diff
@@ -693,7 +693,7 @@ def update_note(submission):
                         decision_note = reply
                         break
             note_accepted = decision_note and 'Accept' in decision_note['content']['decision']['value']
-            submission_readers = self.submission_stage.get_readers(self, submission.number, decision_note['content']['decision']['value'] if decision_note else None, add_publication_chair=self.publication_chair)
+            submission_readers = self.submission_stage.get_readers(self, submission.number, decision_note['content']['decision']['value'] if decision_note else None, add_publication_chair=self.publication_chairs)
```

Comment:
Instead of adding a new parameter, the function get_readers can figure out if they need to add the publication chairs or note based on decision note and `conference.use_publication_chairs` or `conference.publication_chairs_id`

Reply:
done